packages/nimble-components/src/theme-provider/index.ts

```diff
@@ -8,13 +8,34 @@ import { Direction } from '@microsoft/fast-web-utilities';
 import { template } from './template';
 import { styles } from './styles';
 import { Theme } from './types';
+import { pageLocale } from '../utilities/models/page-locale';
+import type { ValidityObject } from '../utilities/models/validator';
 
 declare global {
     interface HTMLElementTagNameMap {
         'nimble-theme-provider': ThemeProvider;
     }
 }
 
+function isInvalidLang(value: string): boolean {
+    try {
+        // eslint-disable-next-line no-new
+        new Intl.DateTimeFormat(value);
```

Comment:
Based on [these MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat), it doesn't look like you need to use the `new` keyword here. Could you avoid disabling the eslint rule by writing this like:

```suggestion
        Intl.DateTimeFormat(value);
```